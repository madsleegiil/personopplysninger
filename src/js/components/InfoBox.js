import React from 'react';

const InfoBox = () => (
  <React.Fragment>
    <div className="BoxContainer">
      <div className="InfoBox">
        <hr width="32px" />
        <h2>Informasjon om familierelasjoner</h2>
        <div className="info-content">
          Denne informasjonen henter Nav for å kunne beregne hvilke ytelser og hvor mye du kan ha
          krav på. for å kunne beregne hvilke ytelser og hvor mye du kan ha for å kunne beregne hvilke ytelser og hvor mye du krav på.

          Les mer om Presonvern erklæringen her
        </div>
      </div>
    </div>
  </React.Fragment>
);

export default InfoBox;
