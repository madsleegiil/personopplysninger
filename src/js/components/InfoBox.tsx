import React from "react";
import PropTypes from "prop-types";

const InfoBox = (props: any) => {
  const { children } = props;
  return (
    <React.Fragment>
      <div className="BoxContainer">
        <div className="InfoBox">
          <hr style={{ width: "32px" }} />
          {children}
        </div>
      </div>
    </React.Fragment>
  );
};

InfoBox.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ])
};

InfoBox.defaultProps = {
  children: []
};

export default InfoBox;
