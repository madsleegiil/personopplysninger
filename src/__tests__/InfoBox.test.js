import * as React from "react";
import wrapIntl from "js/IntlTestHelper";
import InfoBox from "js/components/InfoBox";

const ReactTestRenderer = require("react-test-renderer");

test("render InfoBox with content", () => {
  const component = ReactTestRenderer.create(
    wrapIntl(
      <InfoBox>
        <h2>Header</h2>
        <div className="info-content">Innhold</div>
      </InfoBox>
    )
  );
  expect(component.toJSON()).toMatchSnapshot();
});
