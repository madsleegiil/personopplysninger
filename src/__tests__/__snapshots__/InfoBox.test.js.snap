// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render InfoBox with content 1`] = `
<div
  className="BoxContainer"
>
  <div
    className="InfoBox"
  >
    <hr
      width="32px"
    />
    <h2>
      Header
    </h2>
    <div
      className="info-content"
    >
      Innhold
    </div>
  </div>
</div>
`;
